src/components/dialog/partial-styles/show-hide-animations.scss

```diff
@@ -0,0 +1,34 @@
+$initial-transform: translate3d(0, -1rem, 0) scale(0.9);
+
+dialog {
+    transition:
+        opacity 0.2s ease,
+        transform 0.3s ease;
+    transition-behavior: allow-discrete;
+    opacity: 0;
+    transform: $initial-transform;
+
+    &[open] {
+        opacity: 1;
+        transform: translate3d(0, 0, 0) scale(1);
+
+        @starting-style {
+            opacity: 0;
+            transform: $initial-transform;
+        }
```

Comment:
this is supposed to make sure that the animation applies both when the dialog is opened and closed, but for some reason, every time I have tried this new thing, it only works one-way. 

a good tutorial about this can be found here: 

https://www.youtube.com/watch?v=vmDEHAzj2XE&t=231s